src/main/java/de/fearnixx/jeak/service/controller/InvokationBeforeInitializationException.java

```diff
@@ -0,0 +1,20 @@
+package de.fearnixx.jeak.service.controller;
```

Comment:
I would either rename the exception to something along the lines of `UninitializedControllerException` or move this exception into higher levels of the packages as its name is fairly generic.

Reply:
Which directory would you suggest? I wasn`t able to find any generic Exception directory.

Reply:
Because I always ended up specializing the exceptions so that they don't fit as generic anymore.  
Out off my head, I would use ``de.fearnixx.jeak.except`` or ``de.fearnixx.jeak.service.except``.